terraform/browser/add_response_headers.js

```diff
@@ -4,11 +4,6 @@ function handler(event) {
 
     // Set HTTP security headers
     // Since JavaScript doesn't allow for hyphens in variable names, we use the dict["key"] notation 
-    headers['strict-transport-security'] = { value: 'max-age=63072000; includeSubdomains; preload'}; 
-    // headers['content-security-policy'] = { value: "default-src 'none'; img-src 'self'; script-src 'self'; style-src 'self'; object-src 'none'"}; 
-    // headers['x-content-type-options'] = { value: 'nosniff'}; 
-    // headers['x-frame-options'] = {value: 'DENY'}; 
-    // headers['x-xss-protection'] = {value: '1; mode=block'}; 
 
     // Return the response to viewers 
     return response;
```

Comment:
Lastly, and assuming the removed line is intentional, since this function doesn't do anything anymore, it should be disabled.